### ISSUE TITLE
Separation of site-specific Slurm info (especially UAntwerpen)

### ISSUE THREAD

**Issue by MaximeVdB** [hpcleuven/VscDocumentation#385]
Hi @backelj @RVerschoren (and others from UA):

I would like to clean up / improve the Slurm-related pages. One thing that catches the eye is that (probably for historic reasons) there is quite a bit of UAntwerpen-specific info on the following pages:

https://docs.vscentrum.be/jobs/job_submission.html
https://docs.vscentrum.be/jobs/job_types.html
https://docs.vscentrum.be/jobs/job_advanced.html

I think it would be better to move such info to somewhere in [UAntwerp Tier-2 Infrastructure](https://docs.vscentrum.be/antwerp/tier2_hardware.html), possibly as part of [2.1.5. UAntwerp-specific software instructions :: Slurm workload manager](https://docs.vscentrum.be/antwerp/uantwerp_software_specifics.html#slurm-workload-manager). What do you think? If you're in favor but in lack of time, I can open a PR for this.

**Comment by backelj** [comment]
@MaximeVdB I fully agree — these pages were initially copied from the documentation pages we had under our UAntwerp Tier-2 pages, when they were moved to their now global section Jobs (Running jobs in Slurm). As such, we will have to check all central and Slurm related pages and remove everything that is site-related, but keeping and possibly also adding things that do work on all sites.

In fact, we discussed this yesterday and we are planning on doing this in the next couple of weeks (before our new introductory courses). But feel free to start with it already, and let us know how we can contribute (via a PR or branch or ...).
